index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Sharon Odhiambo</title>
+</head>
+
+<body class="header-background">
+  <header class="nav">
+    <a href="#" id="logo">My logo</a>
+    <nav>
+      <ul class="nav-menu" id="nav-menu">
+        <li class="nav-link"><a href="#works">Portfolio</a></li>
+        <li class="nav-link"><a href="#about-myself">About</a></li>
+        <li class="nav-link"><a href="#form-section">Contact</a></li>
+      </ul>
+      <div class="mobile-navbar" id="mobile-navbar">
+        <div id="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+        <ul class="mobile-nav-menu" id="mobile-nav-menu">
+          <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+          <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
+          <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section class="headline" id="headline"> 
+    <img src="./images/header-bg.svg" alt="Header bg">
```

Comment:
- [ ] Please use CSS background-image property here instead of an `img` tag, because this image is only for decoration (decoration images are used as a background image 👍 ). In the style.css file, add this property to the corresponding selector and make sure it looks like the design provided.
```
background-image: url('./images/header-bg.svg');
```